_data/istio.yml

```diff
@@ -1 +1,2 @@
 version: 0.3.0
+latest_blog_post: /blog/posts/2017/mixer-spof-myth.html
```

Comment:
add the missing newline?

Reply:
Whoops, that was a failed experiment, so I just reverted the change completely.